machine.go

```diff
@@ -217,10 +217,18 @@ func NewMachineWithBackend(backendName string) (*Machine, error) {
 	if _, err := os.Stat("/etc/ca-certificates"); err == nil {
 		m.AddVolume("/etc/ca-certificates")
 	}
+	if _, err := os.Stat("/etc/pki"); err == nil {
+		m.AddVolume("/etc/pki")
+	}
 	if _, err := os.Stat("/etc/ssl"); err == nil {
 		m.AddVolume("/etc/ssl")
 	}
 
+	// Mounts for gnutls (used by wget).
+	if _, err := os.Stat("/etc/crypto-policies"); err == nil {
```

Comment:
crypto-policies is not just for gnutls it's the centralized policies for all crypto :)